chapter2/tests/t4/chapter2-l2-t4-files-no-extensions.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
something is not working here:

![image](https://user-images.githubusercontent.com/26448672/230020254-ce7645ba-b8a0-481b-bab6-a4fac6cb93a3.png)

![image](https://user-images.githubusercontent.com/26448672/230020432-33530ea7-4ec9-4fda-b827-ffbaf2d733ae.png)

Reply:
@kseniyashaydurova 
All fixed now

Reply:
Now there is missed `.test` folder and `file` 
![image](https://user-images.githubusercontent.com/26448672/230600054-b24d6a66-2ee9-413e-a02c-cd0ff7cbbacb.png)

Reply:
> Now there is missed `.test` folder and `file` ![image](https://user-images.githubusercontent.com/26448672/230600054-b24d6a66-2ee9-413e-a02c-cd0ff7cbbacb.png)

fixed